.storybook/preview.tsx

```diff
@@ -21,16 +22,18 @@ export const parameters = {
   },
   darkMode: {
     dark: { ...themes.dark },
-    light: { ...themes.light }
+    light: { ...themes.light },
+    stylePreview: true
```

Comment:
This applies the dark | light classname to the preview iframe `html` tag which then applies the correct styles.